### ISSUE TITLE
[Component] [tooltip] Cannot disable or handle tooltip animation

### ISSUE THREAD

**Issue by sanova** [element-plus/element-plus#14803]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.47`
- Element Plus Version: `2.4.2`
- Browser / OS: `all`
- Build Tool: `Vite`

### Related Component

- `el-tooltip`

### Reproduction Link

[CodeSandbox](https://codesandbox.io/s/element-plus-spa-forked-9kqx82?file=/src/App.vue)

### Steps to reproduce

Moving mouse over list of elements.

### What is Expected?

That tooltip disappears during mouse moving over elements and until it is pointed on a specific element, or at least the transition delay can be set to 0. 

### What is actually happening?

Tooltip has been duplicated many times during mouse moving over elements.

### Additional comments

I'm trying to override animation with custom class (as shown in the example) but it does not work, and there is no information in the docs on how this can be done.
Section "Animation" in the official guide has no related information.

**Comment by Ram4GB** [comment]
How about using `show-after` and setting that prop to `250`? @sanova 

Here is example:
```
<el-tooltip
  content="test tooltip"
  placement="right"
  transition=""
  class="tooltip-no-animation"
  show-after='250'
>
  <div>test tooltip</div>
</el-tooltip>
```

The tooltip won't appear until you wait for it after the specified millisecond.

**Comment by CherishTheYouth** [comment]
> How about using `show-after` and setting that prop to `250`? @sanova
> 
> Here is example:
> 
> ```
> <el-tooltip
>   content="test tooltip"
>   placement="right"
>   transition=""
>   class="tooltip-no-animation"
>   show-after='250'
> >
>   <div>test tooltip</div>
> </el-tooltip>
> ```
> 
> The tooltip won't appear until you wait for it after the specified millisecond.

It works well with your solution

**Comment by sanova** [comment]
Yes you are right. I focused my attention to the wrong attribute "transition" and all other solution and i didn't see that parameter, so easy and i so stypid.
It works very well.
Thank you so much and sorry for my mistake.

**Comment by cuongle-hdwebsoft** [comment]
I'm glad that you can. This is a common issue for everybody who's not familiar with a new library. Don't worry! You have your best idea but we just find a better solution. If this is done, you can close this issue now. Cheer 🍻 @sanova

**Comment by Ram4GB** [comment]
I'm glad that you can. This is a common issue for everybody who's not familiar with a new library. Don't worry! You have your best idea but we just find a better solution. If this is done, you can close this issue now. Cheer 🍻 @sanova

**Comment by Dsaquel** [comment]
I assume the question was answered, closing the issue.
